pkg/acceptance/compose_test.go

```diff
@@ -25,16 +25,16 @@ import (
 
 const composeDir = "compose"
 
-func TestComposeGSS(t *testing.T) {
-	testCompose(t, filepath.Join("gss", "docker-compose.yml"), "psql")
-}
-
-func TestComposeGSSPython(t *testing.T) {
-	testCompose(t, filepath.Join("gss", "docker-compose-python.yml"), "python")
-}
-
-func TestComposeFlyway(t *testing.T) {
-	testCompose(t, filepath.Join("flyway", "docker-compose.yml"), "flyway")
+func TestCompose(t *testing.T) {
```

Comment:
Nit: Is this not a regression from what we had before?

Reply:
Indeed, I did this because all three tests combined take less than a minute. The result we get from dealing with the issue to allow them to run concurrently (propagating a randomly generated id to docker-compose/docker and re-building images to accept that and testing, etc..) is not worth the time/effort we will spend on it.